lucille-core/src/main/java/com/kmwllc/lucille/connector/jdbc/DatabaseConnector.java

```diff
@@ -256,15 +294,15 @@ private String[] getColumnNames(ResultSet rs) throws SQLException {
     return names;
   }
 
-  private void runSql(String sql) {
+  private void runSql(Connection connection, String sql) {
     if (!StringUtils.isEmpty(sql)) {
       try {
         Statement state = connection.createStatement();
         state.executeUpdate(sql);
         state.close();

```

Comment:
If executeUpdate throws an exception the statement won't be closed. The statement can be put inside a try-with-resources.

Reply:
it's in a try with resources now.